src/test_typing.py

```diff
@@ -1928,7 +1932,9 @@ def test_namedtuple_keyword_usage(self):
         self.assertEqual(nick.name, 'Nick')
         self.assertEqual(LocalEmployee.__name__, 'LocalEmployee')
         self.assertEqual(LocalEmployee._fields, ('name', 'age'))
-        self.assertEqual(LocalEmployee._field_types, dict(name=str, age=int))
+        self.assertEqual(LocalEmployee.__annotations__,
+                         collections.OrderedDict(name=str, age=int))
```

Comment:
Maybe it makes sense to use an ordinary ``dict`` in one of the tests? The equality should still be true.